apps/connect/src/components/router.tsx

```diff
@@ -34,13 +31,6 @@ function createRoutes() {
     const routes: RouteObject[] = [
         {
             index: true,
-            element: (
```

Comment:
we can still render the same content for the home page as before, but by treating this route as the main route, we are able to maintain the context for the atoms.